custom_components/powercalc/data/innr/BY 285 C/model.json

```
@@ -0,0 +1,13 @@
+{
+    "calculation_strategy": "lut",
+    "measure_description": "Measured with utils/measure script",
+    "measure_device": "Shelly, 1PM",
+    "measure_method": "script",
+    "measure_settings": {
+        "SAMPLE_COUNT": 2,
+        "SLEEP_TIME": 3,
+        "VERSION": "v1.9.0:docker"
+    },
+    "name": "BY 285 C",
```

```suggestion
    "name": "BY 285 C - 806lm E27 RGBWW",
```

Something like this maybe which is in line with most of the other INNR lights in the library.

Yes, the folder name is used for the auto discovery matching, and optionally one or more `aliases` provided in model.json.

The name is only used in the listing.

---

Makes sense, thanks! Commited